core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.spec.ts

```diff
@@ -263,6 +263,20 @@ describe('Topics and skills dashboard page component', () => {
     expect(componentInstance.filterBoxIsShown).toBeTrue();
   });
 
+  it('should display filter box on maximizing the window', () => {
+    componentInstance.filterBoxIsShown = false;
+    spyOn(windowDimensionsService, 'isWindowNarrow').and.returnValue(false);
+    componentInstance.filterBoxOnResize();
+    expect(componentInstance.filterBoxIsShown).toBeTrue();
+  });
+
+  it('should hide filter box on minimizing the window', () => {
+    componentInstance.filterBoxIsShown = true;
+    spyOn(windowDimensionsService, 'isWindowNarrow').and.returnValue(true);
+    componentInstance.filterBoxOnResize();
+    expect(componentInstance.filterBoxIsShown).toBeFalse();
```

Comment:
```suggestion
    componentInstance.filterBoxIsShown = true;

    spyOn(windowDimensionsService, 'isWindowNarrow').and.returnValue(true);

    componentInstance.filterBoxOnResize();

    expect(componentInstance.filterBoxIsShown).toBeFalse();
```

Reply:
Why do we need a line break after each statement?

Reply:
@divyanshkhetan We have a line break after each statement so that other devs can differentiate between:

1. setup
2. pre-check
3. action
4. post-check

sections of the code.